features/step_definitions/my_google_steps.rb

```diff
@@ -1,82 +1,46 @@
 #These step definitions use Regular Expression matching to perform particular actions when called from a step in the .feature file
 
 Given(/^I am on the Google Home Page$/) do
-  pending # express the regexp above with the code you wish you had
-  #Hint: https://github.com/jnicklas/capybara#navigating
-
-  #SitePrism Hint: Get an instance of Google, then call the method that gives you an instance of GoogleHome, and load it
+  $app = Google.new
+  $app.home.load
 end
 
 When(/^I search for Kittens$/) do
-  pending # express the regexp above with the code you wish you had
-  #Hint: find a good way to identify the search field, enter Kittens in it
-  #Hint: find a good way to identify the search button, click it
-  #Hint: get good at using CSS selectors, which can be used with capybara's "find" method.
-  #Ex. find('#some_element_id').set 'text'
-  #http://www.w3schools.com/cssref/css_selectors.asp
-  #https://github.com/jnicklas/capybara#interacting-with-forms
-  #https://github.com/jnicklas/capybara#clicking-links-and-buttons
-
-  #SitePrism Hint: Using an instance of GoogleHome, perform a search
+  $app.home.set_search_field('Kittens')
+  $app.home.click_search_button
 end
 
 Then(/^I see the Google Search Results Page$/) do
-  pending # express the regexp above with the code you wish you had
-  #Hint: how do you know you're on a different page than before?
-  #https://github.com/jnicklas/capybara#navigating
-
-  #Then steps are most often going to involve RSpec expectation matchers
-  #Which take this form
-  #expect(current_url).to contain(expected_string)
-  #expect(current_url).to match(/\?some_param=\d+/)
-  #https://relishapp.com/rspec/rspec-expectations/docs
-
-  #SitePrism Hint: Get an instance of GoogleSearchResults. If your GoogleSearchResults class has set_url_matcher, how can you assert that the page url is loaded?
-  #SitePrism Hint: If your GoogleSearchResults class has elements defined, how can you use one assert to ensure all defined elements are present?
+  $app.search_results.load(query: 'Kittens')
```

Comment:
If this step comes after you click the search button, should you have to actually load the search results page? This step should pass with just asserting:
```ruby
expect($app.search_results).to be_displayed
#or
expect($app.search_results.displayed?).to be_truthy
```